bedrock/newsletter/views.py

```diff
@@ -610,7 +610,7 @@ def newsletter_subscribe(request):
             if 'email' in form.errors:
                 errors.append(_('Please enter a valid email address'))
             if 'privacy' in form.errors:
-                errors.append(_('You must agree to the privacy policy'))
+                errors.append(_('You must agree to the privacy notice'))
```

Comment:
We could do an l10n tag here since we have access to the `request` object and thus can know what locale we're rendering.